### DISCUSSION TITLE
GP2040 and Brook Wingman

### DISCUSSION THREAD

**Discussion by mklinik** [OpenStickCommunity/GP2040-CE#303] [General]
Just a thought, could the Brook Wingman be used to make it compatible with PS5 games? https://www.brookaccessory.com/products/wingmanfgc/index.html

The Brook Wingman is an adapter that let's PS4 fight sticks work with PS5 games.

**Comment by bsstephan** [comment]
I haven't received my Wingman FGC yet, but other folks in the community have and it apparently works great. But, note that it only makes sticks compatible with PS5 *fighting games*, not all PS5 games.
